sources/packages/backend/apps/queue-consumers/test/helpers/testing-modules/testing-modules-helper.ts

```diff
@@ -17,6 +18,7 @@ import { ClamAVService, SystemUsersService, ZeebeModule } from "@sims/services";
 import { ZeebeGrpcClient } from "@camunda8/sdk/dist/zeebe";
 import { createCASServiceMock } from "../mock-utils/cas-service.mock";
 import { CASService } from "@sims/integrations/cas/cas.service";
+import { ObjectStorageService } from "@sims/integrations/object-storage";
```

Comment:
Please use the index.

Reply:
Didn't get it. This points to the index.